src/liger_kernel/ops/fused_linear_cross_entropy.py

```diff
@@ -54,7 +55,25 @@ def fused_linear_cross_entropy_forward(
     loss_1d = torch.zeros(BT, dtype=torch.float32, device=device)
 
     # NOTE: skip .item() here to avoid CUDA synchronization
-    total_n_non_ignore = (target != ignore_index).sum()
+    target_mask = target != ignore_index
+    total_n_non_ignore = target_mask.sum().item()
```

Comment:
I have noticed the comment above to avoid using multiple times. Will this change align the synchronization?

Reply:
Forgot to remove the comment, it doesn't affect the result.